### ISSUE TITLE
Translate validator messages

### ISSUE THREAD

**Issue by peterjaap** [deity-io/falcon#501]
**What should be changed/improved?**
Translation of validator messages should be added, see TODO note; https://github.com/deity-io/falcon/blob/dev/packages/falcon-ecommerce-uikit/src/Forms/validators.ts#L3

**Comment by peterjaap** [comment]
For now, we'll override the component in `falcon-client.build.config.js` and add the translations directly;

```
...
  moduleOverride: {
    '@deity/falcon-ecommerce-uikit/dist/Forms/validators': './src/components/validators'
  }
...
```

**Comment by tutok** [comment]
resolved #585
